### PR TITLE
Prepare for v0.12

### PR DESCRIPTION
**Pull request by ludwigschwardt**
And the last episode of the season.

### CODE REVIEW

File: NEWS.rst [hunk 1]
```diff
@@ -1,6 +1,16 @@
 History
 =======
 
+0.12 (2019-02-12)
+-----------------
+* Make L1 product by applying calibration corrections aka applycal (#194 - #198)
```
Comment (bmerry) [inline NEWS.rst]: This makes it sounds like it happens automatically, which might lead people astray.

File: NEWS.rst [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+* Let default reference antenna in v4 datasets be "array" antenna (#202, #220)
+* Use katsdptelstate v0.7 features: msgpack, memory backend (#196, #201, #212)
```
Comment (bmerry) [inline NEWS.rst]: We're not using msgpack anywhere, just using the encode/decode helpers.

File: NEWS.rst [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+* Prepare for multi-dump chunks (#213, #214, #216, #217, #219)
+* Allow L1 flags to be ignored (#209, #210)
+* Fix usage of deprecated dask features (#204, #215)
```
Comment (bmerry) [inline NEWS.rst]: I would say *some* deprecated dask features, since we're still using sharedict.

File: setup.py [hunk 4]
```diff
@@ -18,12 +18,12 @@
 
 from __future__ import print_function, division, absolute_import
 
-import os.path
+import os
```
Comment (bmerry) [inline setup.py]: Why?

File: setup.py [hunk 5]
```diff
@@ -58,8 +58,8 @@
       python_requires='>=2.7,!=3.0,!=3.1,!=3.2',
       setup_requires=['katversion'],
       use_katversion=True,
-      install_requires=['numpy', 'katpoint', 'h5py>=2.3',
-                        'katsdptelstate[rdb]', 'dask[array] >= 0.18.2', 'numba',
+      install_requires=['numpy', 'katpoint', 'h5py >= 2.3', 'numba',
+                        'katsdptelstate[rdb] >= 0.7', 'dask[array] >= 0.18.2',
```
Comment (bmerry) [inline setup.py]: Does it still work with dask 0.18.2, or are we using any newer features?

File: setup.py (hunk 4 above)
Comment (ludwigschwardt) [reply setup.py]: There is some [confusion](https://stackoverflow.com/questions/2724348) around this. I somehow mistakenly recalled that Python 3 encouraged `os` above `os.path`. I'm happy to change it back - more explicit that way.

Comment (bmerry) [reply setup.py]: I would change it back, but I'm not too fussed.

File: NEWS.rst (hunk 1 above)
Comment (ludwigschwardt) [reply NEWS.rst]: Fixed.

File: NEWS.rst (hunk 2 above)
Comment (ludwigschwardt) [reply NEWS.rst]: Sure.

File: NEWS.rst (hunk 3 above)
Comment (ludwigschwardt) [reply NEWS.rst]: Reworded.

File: setup.py (hunk 4 above)
Comment (ludwigschwardt) [reply setup.py]: Changed back.

File: setup.py (hunk 5 above)
Comment (ludwigschwardt) [reply setup.py]: I can't see any newer features. I am still keen to resolve the `HighLevelGraph` warning soon for the record, as I don't like scaring the users.

**Pull request merged**